main.go

```diff
@@ -133,8 +134,13 @@ func main() {
 		log.Panicf("connect mongo database with error: %s", err)
 	}
 
+	acct, err := account.FromSeed(viper.GetString("account.seed"))
+	if err != nil {
+		log.Panic(err)
+	}
+
 	// Init http server
-	server = web.NewServer(mongoClient)
+	server = web.NewServer(viper.GetString("macaroon.location"), mongoClient, acct.(*account.AccountV2))
```

Comment:
```suggestion
	server = web.NewServer(viper.GetString("server.endpoint"), mongoClient, acct.(*account.AccountV2))
```
or `server.hostname`